test 10/README.txt

```diff
@@ -0,0 +1 @@
+This test makes sure that a sale that would make a quantity go below 0 doesn't run
```

Comment:
this test checks transition from 0 to -x, and arrival to 0. there are 2 test in one here (: but very good